Atomic/util.py

```diff
@@ -124,7 +123,8 @@ def subp(cmd, cwd=None, newline=False):
     proc = subprocess.Popen(cmd, cwd=cwd,
                             stdout=subprocess.PIPE,
                             stderr=subprocess.PIPE, close_fds=True,
-                            universal_newlines=newline)
+                            universal_newlines=newline,
+                            env=os.environ)
```

Comment:
When we call skopeo, we need to pass the proxy information which is done via environment variables.